roboflow/core/version.py

```diff
@@ -246,7 +232,7 @@ def download(self, model_format=None, location=None, overwrite: bool = False):
 
         self.__download_zip(link, location, model_format)
         self.__extract_zip(location, model_format)
-        self.__reformat_yaml(location, model_format)
+        self.__reformat_yaml(location, model_format)  # TODO: is roboflow-python a place to be munging yaml files?
```

Comment:
I speculate, based on 2 days of experience with this:

In the past - yes. `data.yml` used to contain `path`, providing a path to the dataset. This would be based on where the dataset is located on the system. Now, yolov8 and yolo11 do not require that, but I don't know about other models.

Reply:
This will not fix the path attribute. It will leave the train/test/val attributes untouched with the original content that was downloaded.
There's [this other PR](https://github.com/roboflow/Annotation.js/pull/119) on Annotation.js that will remove that.